src/QueueReceiver.Core/Services/AccessService.cs

```diff
@@ -11,18 +11,18 @@ namespace QueueReceiver.Core.Services
     public class AccessService : IAccessService
     {
         private readonly IPersonService _personService;
-        private readonly IProjectService _projectService;
+        private readonly IPersonProjectService _personProjectService;
```

Comment:
This change causes compliationErrors further down the file. Looks like a partial refactor.